src/Sylius/Component/Core/Repository/OrderRepositoryInterface.php

```diff
@@ -69,6 +69,13 @@ public function findForCustomerStatistics(CustomerInterface $customer);
     public function findOneForPayment($id);
 
     /**
+     * @param int $id
```

Comment:
Really? Almost everywhere we have `int` next to `$id`, also in this repository class. If this is a mistake, we should refactor docblocks in a separate PR.